code/obj/machinery/manufacturer.dm

```diff
@@ -1287,36 +1268,30 @@
 				src.malfunction = !src.malfunction
 			if (WIRE_POWER)
 				if(!(src.status & BROKEN || src.status & NOPOWER))
-					src.manuf_zap(usr,100)
+					src.shock(usr,100)
```

Comment:
same thing as the other comments about `usr` (which are somewhere far up the page by now)